packages/hooks/src/Provider.tsx

```diff
@@ -46,15 +48,50 @@ export const Provider: React.FC<ProviderProps> = ({ children, network, infuraId
   const [connected, setConnected] = React.useState<boolean>(false);
 
   const connectWallet = React.useCallback(async () => {
+    // Coinbase walletLink init
+    const walletLink = new WalletLink({
+      appName: 'coinbase',
+    });
+
+    // WalletLink provider
+    const walletLinkProvider = walletLink.makeWeb3Provider(
+      'https://eth-mainnet.alchemyapi.io/v2/qCdzfF9UqXcJYIle-Ff-BN0MII8LjLQs',
+      1
+    );
+
     const web3Modal = new Web3Modal({
       network,
       providerOptions: {
         walletconnect: {
           package: WalletConnectProvider,
           options: {
+            bridge: 'https://polygon.bridge.walletconnect.org',
             infuraId,
+            rpc: {
+              1: 'https://eth-mainnet.alchemyapi.io/v2/qCdzfF9UqXcJYIle-Ff-BN0MII8LjLQs', // mainnet // For more WalletConnect providers: https://docs.walletconnect.org/quick-start/dapps/web3-provider#required
+              42: infuraId,
+              100: 'https://dai.poa.network', // xDai
+            },
           },
         },
+        'custom-walletlink': {
+          display: {
+            logo: 'https://play-lh.googleusercontent.com/PjoJoG27miSglVBXoXrxBSLveV6e3EeBPpNY55aiUUBM9Q1RCETKCOqdOkX2ZydqVf0',
```

Comment:
let's grab a coinbase svg from somewhere and move it into the repo, that way we don't need to rely on a random google url

Reply:
Sure, I'll do it that way.